<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationPopup</class>
 <widget class="QWidget" name="AnimationPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="fpsSlider">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ALabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>Loading...</string>
   </property>
  </widget>
  <widget class="QLabel" name="minFPSLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="maxFPSLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>41</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>24</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="actualSizeSelector">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Actual Size</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentFPSLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>41</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
